.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    "env": {
+        "commonjs": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-console": 0,
```

Comment:
Why disable errors when using `console`? Doesn't seem like we ever want console in the code... that's what `debug` is for.